<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCalibFromLinesConfig</class>
 <widget class="QWidget" name="CCalibFromLinesConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="downsample_factor_label">
       <property name="text">
        <string>Downsampling factor:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="downsample_factor_sbox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="segment_label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Line Segmentation&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <spacer name="label_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="clow_threshold_label">
       <property name="text">
        <string>Canny Low Threshold:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="clow_threshold_sbox">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>150</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="chighlowratio_label">
       <property name="text">
        <string>Canny High to Low Ratio:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="chightolow_ratio_sbox">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>5</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="ckernel_size_label">
       <property name="text">
        <string>Canny Kernel Size:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="ckernel_size_sbox">
       <property name="minimum">
        <number>3</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>12</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="hthreshold_label">
       <property name="text">
        <string>Hough Threshold:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="hthreshold_sbox">
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>150</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="button_group">
     <item>
      <widget class="QPushButton" name="extract_lines_button">
       <property name="text">
        <string>Extract Lines</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="match_lines_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Match Lines</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="calib_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Calibrate</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="save_calib_button">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Save Calibration</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="save_params_button">
     <property name="text">
      <string>Save Parameters</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
